ledger-core/v2/cmd/heavy-badger/scan.go

```diff
@@ -181,7 +181,7 @@ func (dbs *dbScanner) getAllPulses() (pulses []insolar.Pulse) {
 	for ; err == nil; p, err = pulseStore.Backwards(ctx, p.PulseNumber, 1) {
 		pulses = append(pulses, p)
 	}
-	if err != nil && err != pulsedb.ErrNotFound {
+	if err != pulsedb.ErrNotFound {
```

Comment:
If err will be != ErrNotFound , err = SomeError for example, fatalf will not happen, is it ok?